### ISSUE TITLE
Any change to Global links doesn't save.

### ISSUE THREAD

**Issue by Joka19** [CharlesGameDev/Fushigi#24]
If I add or delete a global link and press save, the next time when I load the same level, the changes are gone.

Any way to fix that?

**Comment by CharlesGameDev** [comment]
Fixed in v1.1.5.1

**Comment by CharlesGameDev** [comment]
I lied

**Comment by Mariosonicman91** [comment]
Yeah, having this exact issue editing 1-2. Can't link the pipe a new pipe to the one in the background. Good to know I'm not going crazy. GL, CharlesGameDev! Hope you solve this bug!

**Comment by Joka19** [comment]
Update on this problem?
